### PR TITLE
Document list - Create Folder Docs

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

No documentation about how to enable and disable the creation menu

**What is the new behavior?**

The documentation is done

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: ng2-components/ng2-alfresco-documentlist/src/components/document-list.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
-<alfresco-document-menu-action *ngIf="contentMenuActions"
-        [currentFolderPath]="currentFolderPath"
-        (onSuccess)="reload()">
+<alfresco-document-menu-action *ngIf="creationMenuActions"
+                               [currentFolderPath]="currentFolderPath"
+                               (onSuccess)="onActionMenuSuccess($event)"
+                               (onError)="onActionMenuError($event)">
```
Comment (VitoAlbano) [inline ng2-components/ng2-alfresco-documentlist/src/components/document-list.html]: this should be (error)

**Comment by magemello** [comment]
can we add more documentation like and a nice image of how it looks?
see content-action section as example

File: ng2-components/ng2-alfresco-documentlist/src/components/document-list.html (hunk 1 above)
Comment (magemello) [inline ng2-components/ng2-alfresco-documentlist/src/components/document-list.html]: ok seems that Vito is right (error) and (success) are the best options

Comment (magemello) [inline ng2-components/ng2-alfresco-documentlist/src/components/document-list.html]: ok seems that Vito is right (error) and (success) are the best options

**Pull request merged**